src/hrflow_connectors/connectors/destinations/smartrecruiters/actions.py

```diff
@@ -0,0 +1,151 @@
+
+            formated_project_dict["startDate"] = start_date
+
+            end_datetime_str = project.get("date_end")
+            if end_datetime_str is None:
+                end_date = "XXXX"
+            else:
+                end_date = end_datetime_str.split("T")[0]
+
+            formated_project_dict["endDate"] = end_date
+            formated_project_dict["location"] = value_or_empty(
+                project["location"]["text"]
+            )
+            formated_project_dict["description"] = project["description"]
+
+            return formated_project_dict
+
+        def format_educations(educations):
+            formated_education_list = []
+
+            for education_entity in educations:
+                formated_education = format_project(education_entity)
+                if education_entity.get("school") is None:
+                    formated_education["instituion"] = "NaN"
+                else:
+                    formated_education["institution"] = education_entity.get("school")
+
+                if education_entity.get("title") is None:
+                    formated_education["degree"] = "Nan"
+                else:
+                    formated_education["degree"] = education_entity.get("title")
+                formated_education["major"] = "NaN"
+
+                formated_education_list.append(formated_education)
+
+            return formated_education_list
+
+        def format_experiences(experiences):
+            formated_experience_list = []
+
+            for exp in experiences:
+                formated_exp = format_project(exp)
+                if exp["title"] is None:
+                    formated_exp["title"] = "NaN"
+                else:
+                    formated_exp["title"] = exp["title"]
+                if exp["company"] is None:
+                    formated_exp["company"] = "NaN"
+                else:
+                    formated_exp["company"] = exp["company"]
+
+                formated_experience_list.append(formated_exp)
+
+            return formated_experience_list
+
+        info = profile["info"]
+        smart_candidate = dict()
+        smart_candidate["firstName"] = info["first_name"]
+        smart_candidate["lastName"] = info["last_name"]
+        smart_candidate["email"] = info["email"]
+        smart_candidate["phoneNumber"] = info["phone"]
+
+        if info["location"]["fields"] not in [
+            [],
+            None,
+        ]:  # check if fields is not an empty list
+            smart_candidate["location"] = dict(
+                city=value_or_empty(info["location"]["fields"].get("city", {})),
+                country=value_or_empty(info["location"]["fields"].get("country", {})),
+                region=value_or_empty(info["location"]["fields"].get("state", {})),
+                lat=info["location"]["lat"] if info["location"]["lat"] else 0,
+                lng=info["location"]["lng"] if info["location"]["lng"] else 0,
```

Comment:
`is not None` plutôt.
Sinon, on peut avoir
```python
if 0.0:
  print("pas affiché")
else:
  print("affiché")
```
Car `0.0` représente `false`. (de même que `""`)